src/VisualStudio/Core/Def/Implementation/LanguageClient/LiveShareInProcLanguageClient.cs

```diff
@@ -42,6 +43,24 @@ internal class LiveShareInProcLanguageClient : AbstractInProcLanguageClient
         public override string Name => "Live Share C#/Visual Basic Language Server Client";
 
         protected internal override VSServerCapabilities GetCapabilities()
-            => _defaultCapabilitiesProvider.GetCapabilities();
+        {
+            var experimentationService = Workspace.Services.GetRequiredService<IExperimentationService>();
```

Comment:
Note this change may be depending on https://github.com/dotnet/roslyn/pull/51295 for correctness; just be careful/aware here.